packages/lesswrong/components/review/ReviewVotingPage.tsx

```diff
@@ -448,92 +446,6 @@ const ReviewVotingPage = ({classes}: {
     reSortPosts(sortPosts, sortReversed)
   }, [canInitialResort, reSortPosts, sortPosts, sortReversed])
 
-  const FaqCard = ({linkText, children}) => (
-    <LWTooltip tooltip={false} clickable title={
-      <Card className={classes.faqCard}>
-        <ContentStyles contentType="comment">
-          {children}
-        </ContentStyles>
-      </Card>}
-    >
-      {linkText}
-    </LWTooltip>
-  )
-  
-  const instructions = isEAForum ?
```

Comment:
I'm removing this because I expect to have refactored the descriptions pretty significantly by the time EA Forum next hosts a review, and if EA Forum hosts a review again it'll make more sense to tweak the wording of whatever I end up with than keep this exactly.